inc/namespace.php

```diff
@@ -84,7 +84,7 @@ function bootstrap() {
 	add_filter( 'wp_mail_from', function () use ( $config ) : string {
 		$default_email = sprintf(
 			'noreply@%s',
-			wp_parse_url( network_home_url( '/' ), PHP_URL_HOST )
+			str_replace( 'www.', '', wp_parse_url( network_home_url( '/' ), PHP_URL_HOST ) )
```

Comment:
This will replace it anywhere in the URL, so if we had a client like `bestofwww.com` it'd ruin the domain.

I think actually though, using something like this may be better:
```php
$domain = get_network()->domain;
if ( substr( $domain, 0, 4 ) === 'www. ) {
    $domain = substr( $domain, 4 );
}
```

No need to generate a full URL.